data/categories.txt

```diff
@@ -7319,6 +7350,8 @@ fa:گاراژ
 
 highway-rest_area
 en:Highway Rest Area
+en-GB:Service Station|Motorway Services
```

Comment:
This is not a service station: https://wiki.openstreetmap.org/wiki/Tag:highway%3Drest_area

Reply:
You're right! Fixed.